src/components/charts/RangeBar.js

```diff
@@ -0,0 +1,100 @@
+import React from "react";
```

Comment:
RangeBars should handle `NaN` better, e.g. "Le Hunte - Elliston":

<img width="292" alt="Screen Shot 2021-06-21 at 12 16 51 PM" src="https://user-images.githubusercontent.com/244952/122795411-4a135480-d28b-11eb-901d-83c974d0c68d.png">

Ideally they'd have some way for the range to show the value is undefined, but for now let's just replace the RangeBar with the text "Undefined" when a value is `NaN`.